src/traces/parcoords/calc.js

```diff
@@ -39,3 +45,11 @@ function constHalf(len) {
     }
     return out;
 }
+
+function isTypedArray(a) {
```

Comment:
No need for this. `Lib.isTypedArray` [is](https://github.com/plotly/plotly.js/blob/ecfa45a0f00b1b49c2fda9853e6ac69fa06c7a8f/src/lib/index.js#L29) a [thing](https://github.com/plotly/plotly.js/blob/ecfa45a0f00b1b49c2fda9853e6ac69fa06c7a8f/src/lib/array.js#L23-L26).

Reply:
@etpinard Thanks for the note. 
Done in e577e49.